src/clr-angular/forms/styles/_file.clarity.scss

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@include exports('forms.control.file') {
+    .clr-file-wrapper {
+        @include form-flatten-validate-text();
+        position: relative;
+        margin-top: $clr-forms-baseline;
+
+        .clr-file {
```

Comment:
Can move out of `.clr-file-wrapper`

Reply:
We can, but this does require having a wrapper for it to work.

Reply:
I think thats fine. The advantage of removing the nesting is that people can target `.clr-file` directly instead of having to write `.clr-file-wrapper .clr-file` if they want to customize something.

Reply:
Done.